.github/workflows/nightly-scan.yml

```diff
@@ -29,14 +29,18 @@ jobs:
       - uses: actions/checkout@v2
       - name: Set job environment variables
         run:  cat JOB.env >> $GITHUB_ENV
+
       - name: Scan .NET Core docker image
-        run: |
-          curl -s https://ci-tools.anchore.io/inline_scan-v$ANCHORE_SCAN_VERSION | \
-          bash -s -- -r -f -p -b ./anchore-policy.json defradigital/$IMAGE_NAME:$DEFRA_VERSION-dotnet${{matrix.image.runTimeVersion}}
+        id: scan
+        uses: anchore/scan-action@v3
+        with:
+          image: defradigital/${{env.IMAGE_NAME}}:${{env.DEFRA_VERSION}}-dotnet${{matrix.image.runTimeVersion}}
+          fail-build: true
+          severity-cutoff: "medium"
 
       - name: Archive anchore report
         if: ${{ failure() }}
         uses: actions/upload-artifact@v2
         with:
-          name: anchore-reports-dotnet-${{ matrix.image.runtimeVersion }}
-          path: anchore-reports
+          name:  anchore-reports-dotnet-${{ matrix.image.runtimeVersion }}
```

Comment:
did you want to call this one "netcore" instead of "dotnet" too?

Reply:
Or vice versa! Thanks, good spot on both, I've updated them now.